packages/tupaia-web/src/features/DashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -135,11 +131,12 @@ export const EnlargedDashboardItem = () => {
   // If the report is a drilldown, it will have a drilldown id in the url
   const drilldownId = urlSearchParams.get(URL_SEARCH_PARAMS.REPORT_DRILLDOWN_ID);
   // At this time we only support drilldown in matrix visuals
-  const isDrillDown = currentDashboardItem?.type === 'matrix' && !!drilldownId;
+  const isDrillDown = currentDashboardItem?.config?.type === 'matrix' && !!drilldownId;
   // If the report is a drilldown, we want to get the parent dashboard item, so that we can get the parameter link for querying the data, and also so that we can show a back button to the correct parent dashboard item
   const parentDashboardItem = isDrillDown
     ? activeDashboard?.items.find(
-        report => report.drillDown && report.drillDown.itemCode === reportCode,
+        // @ts-ignore - drillDown is all lowercase in the types config
+        dashboardItem => dashboardItem?.config?.drillDown?.itemCode === reportCode,
```

Comment:
I think you can overwrite the casing as we do with `entityHeader` in the `DashboardItem` type